storage/reads/array_cursor_test.go

```diff
@@ -0,0 +1,2251 @@
+package reads
```

Comment:
@danxmoran is this file the same as 2.x as well?

Reply:
Yes (sorry, added it later). The new mock cursor files are the same too.